tests/test-form-data-boundary.js

```diff
@@ -100,13 +100,35 @@ tape('custom boundary within quotes', function (t) {
   })
 })
 
+tape('content-length without content-type', function (t) {
+  request.post({
+    url: server.url,
+    headers: {
+      'content-length': '171'
+    },
+    formData: {
+      formKey: 'formValue'
+    }
+  }, function (err, res, body) {
+    var req = JSON.parse(body)
+    var boundary
+    t.equal(err, null)
+    t.equal(res.statusCode, 200)
+    t.notEqual(req.headers['content-type'], null)
+    boundary = req.headers['content-type'].split('boundary=')[1]
+    t.equal(req.headers['content-type'], 'multipart/form-data; boundary=' + boundary)
+    t.equal(req.headers['content-length'], '171')
+    t.ok(req.body.indexOf('name="formKey"') !== -1)
```

Comment:
Also test for boundary on `req.body`, like its done in other tests:
```js
t.ok(req.body.startsWith('--' + boundary))
t.ok(req.body.endsWith(boundary + '--\r\n'))
```

Reply:
Added in c9cfe9f